auxin_cli/src/main.rs

```diff
@@ -163,6 +165,91 @@ impl SignalPrekeyData {
 	}
 }
 
+/// Does last steps to fully register a signal account
+async fn fix_accounts<R>(client: &reqwest::Client, account: &SignalAccount<R>) -> Result<()> {
+	// TODO(Diana): This seems to basically just be
+	// AuxinApp::retrieve_profile but on ourselves
+	// But we can't use AuxinApp here
+	let uuid = account.aci().uuid();
+	let version = account.profile_key().version(uuid);
+	let url = format!("https://chat.signal.org/v1/profile/{uuid}/{version}",);
+	let res = client
+		.get(url)
+		.basic_auth(account.aci().uuid(), Some(account.password()))
+		.send()
+		.await?;
+	// TODO(Diana): We don't currently care about the response
+	let _profile: auxin::profile::ProfileResponse = res.json().await?;
```

Comment:
```suggestion
	let _: auxin::profile::ProfileResponse = res.json().await?;
```

Reply:
Does the result need to be driven by an await even? If it is, maybe it can be launched as a task instead of awaiting immediately and the handle can be awaited later when it's absolutely necessary to do so.